### DISCUSSION TITLE
Druid Leveler

### DISCUSSION THREAD

**Discussion by ebner20** [isid0re/SoloLeveling#51] [General]
I have been running the Druid build now for 2 days.  Here is what I have observed

1.  He makes lots and lots of stealth armors lol.  I think its looking for upgrades maybe but when ever it finds a two slotter he creates a new one.
2. Its still at 30 at 40+ hours.  Not sure if its due to gear or what.  He does try to get to Diablo but chickens out of game.  Is there a way that if he figures out that he isn't strong enough to maybe try something else?  I see on my logs that it "run completed" than "restarting script again" but he appears to not improving at all.  Exp bar has been at 1/4 for the past 12 hours.  

I did update to your latest update last night.  No improvement.  

I do have your sorc leveler running on another computer that I started this morning and she is killing it.  Already level 41.  Thats after 10 hours.  So that script appears to be working. I'm just not sure if Druid is.

**Comment by isid0re** [comment]
Hmm level 30 after 2 days? Is it entering act 4 or chickening in act 3 before or after mephisto?

**Reply by ebner20** [reply]
It got to Diablo but is not strong enough to beat him.  So they chicken out.  They did that a few times but it restarts the run.   Pretty slow going but I think its more of the gear being bad than its your script.  I think it needs to get to a certain point damage wise of being able to move forward and for the bot to continue (if that makes any sense at all...) or it just keeps restarting.  I honestly believe this because I ran your Necro on a toon and he made it to 66 (I had a friend drop some gear at around 30 for the first Necro)  before he died on hardcore.  I've since recreated a Necro and he is now stuck at level 30 like this Druid.

**Reply by isid0re** [reply]
I see. I've been looking thru all the scripts for areas to improve speed and safety. I've found a couple that I've been testing out currently. 

One of the biggest time sucks has been the excessive shopping. I'm testing a reduced shopping setup to see if it improves run time as well.

**Reply by ebner20** [reply]
You must have changed something cuz hes moving along now.  Good job.  Loving this a lot.  Actually, I went off your files and made an assassin solo leveler.  I'm sure there is more to change on it but so far its running well and autobuilding. 

If you have time to look it over and maybe correct shit id send it your way.  I stole bits and pieces from different scripts I found on the internet.  Not sure if you have each class searching for specific gear cuz if so, my assassin will be looking for pally stuff lol.

**Reply by isid0re** [reply]
Yea, I'd love to see your assassin builds. Start a new thread and post the builds there.
